### DISCUSSION TITLE
Artificial intelligence is becoming more and more powerful, should we give up programming?

### DISCUSSION THREAD

**Discussion by shaoting0730** [community/community#44457] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Artificial intelligence is becoming more and more powerful, should we give up programming? why?

**Comment by bzkarim250** [comment]
AI is a powerful tool that can help us automate certain tasks and make them more efficient, but it is not a replacement for human intelligence or programming. **The combination of human intelligence and AI will be more powerful,** and the field of AI will continue to evolve, **requiring programmers to develop and maintain new systems**.

I hope this might be a good Idea.

**Comment by engrajibulhasan** [comment]
**AI** is growing up. In future it will be stronger. But in  the meantime more skilled Programmer demand will also be increases and less skilled programmer demand will also be reduce.
There is no chance of Human replacement.

**Reply by bzkarim250** [reply]
exactly

**Comment by chthonn** [comment]
###  **If AI gets stuck somewhere then they need us**

**Comment by mrowan137** [comment]
Prof. Pedro Domingos summed up it as as best I've heard:

"It's not man versus machine; it's man with machine versus man without. Data and intuition are like horse and rider, and you don't try to outrun a horse; you ride it."

Human intuition is a valuable asset, and will be hard to replicate.  AI may change the way people program, but we may view it as a tool that augments our ability to solve challenging problems, rather than a replacement.
